specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -12806,6 +15566,90 @@
         }
       }
     },
+    "ContinuousWebJob": {
+      "description": "Continuous Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ContinuousWebJob resource specific properties",
+          "properties": {
+            "status": {
+              "description": "Job status.",
+              "type": "string"
+            },
+            "detailedStatus": {
+              "description": "Detailed status.",
+              "type": "string"
+            },
+            "logUrl": {
+              "description": "Log URL.",
+              "type": "string"
+            },
+            "name": {
+              "description": "Job name. Used as job identifier in ARM resource URI.",
+              "type": "string"
+            },
+            "runCommand": {
+              "description": "Run command.",
+              "type": "string"
+            },
+            "url": {
+              "description": "Job URL.",
+              "type": "string"
+            },
+            "extraInfoUrl": {
+              "description": "Extra Info URL.",
+              "type": "string"
+            },
+            "jobType": {
+              "description": "Job type.",
+              "type": "string"
+            },
+            "error": {
+              "description": "Error information.",
+              "type": "string"
+            },
+            "usingSdk": {
```

Comment:
what does this property used for? Not very intuitive. May be make it an enum with a better property name. But this would mean service side change on your side.